lib/Config/Migrator.php

```diff
@@ -469,6 +470,20 @@ function log() {
     return $this->sqlify(__FUNCTION__, $attributes);
   }
 
+  function userFlags() {
+    $attributes = array(
+      'id int(11) unsigned NOT NULL AUTO_INCREMENT,',
+      'user_id bigint(20) NOT NULL,',
+      'name varchar(50) NOT NULL,',
+      'value longtext,',
```

Comment:
Do we really need `longtext` here? I think we should even make it boolean. Or if that is not possible it could be `char(1)` We really need to store 1 or 0 if the user has already seen the feature or not.

Reply:
Not all flags are booleans, `last_announcement_seen`, for example, is a number.

Reply:
Ok, that is true. But we still don't need `longtext`. Something smaller would be enough.